src/_includes/layouts/webinar.njk

```diff
@@ -45,7 +45,12 @@ layout: layouts/base.njk
                                 region: "eu1",
                                 portalId: "26586079",
                                 formId: formId,
-                                onFormSubmit: function($form) {
+                                onFormSubmit: function($form, data) {
+                                    // identify user 
+                                    const formData = new Map(data.map((obj) => [obj.name, obj.value]));
```

Comment:
[nb] I'm not sure exactly how HubSpot implements this callback, but there is a native [FormData](https://developer.mozilla.org/en-US/docs/Web/API/FormData/FormData) object now.